community-membership.md

```diff
@@ -296,5 +297,69 @@ with the project. When a contributor returns to being more active, they may be
 promoted back to their previous position at the discretion of the current
 maintainers following the process defined above.
 
+## Specification Sponsor
+
+Specification sponsors are trusted collaborators of the technical committee, and
+work to review, approve, and sponsor [opentelemetry-specification][] issues and
+PRs.
+
+Members are defined
+in [Community Members -> Specification and Protos](./community-members.md#specifications-and-proto)
+and are members of the `spec-sponsors` team (TODO: add link to github team when
+available). Specification sponsors SHOULD list
+their [areas of interest](./areas-of-interest.md).
+
+### Requirements
+
+Specification sponsors are trusted to drive specification work which can be
+complex and which highly impactful to downstream language SIGs and to the
+OpenTelemetry project as a whole. Approvals from specification sponsors count
+towards the approvals required to merge specification PRs. Therefore, there is a
+high bar to becoming a specification sponsor.
+
+- Contributor to OpenTelemetry project for at least 1 year
+- Has been the reviewer or author (through the sponsorship of another
+  specification sponsor) of at least 10 substantial PRs to specification or
+  related repositories
+- Demonstrated sound technical judgement, and a deep understanding of the
+  specification and the OpenTelemetry project
+- Nominated by technical committee member (
+  see [becoming a specification sponsor](#becoming-a-specification-sponsor))
+
+### Responsibilities and privileges
```

Comment:
How many hours per week are we talking about here? If we have an idea, it would be good to share here.

Reply:
I don't have any idea. Same goes for maintainer, approver, triager, and TC membership. The time commitment varies by person. 

There might be a reasonable minimum. Like, its hard to imagine fulfilling the responsiveness responsibility without at least 1-2 hours per week. I would also expect that sponsors do their best to attend the specification SIG.

Reply:
I acknowledge that the language is similar to the other roles, and I'm OK with having this change here look like those. My comments on this review were mostly taking the perspective of someone coming to the community and thinking why would they decide to be a spec sponsor in the first place. Perhaps framing it that way would open a door for changing the other descriptions to be more enticing to new collaborators as well.

Feel free to mark this and my other comments as resolved without changing the PR.